sdk/react/src/Core.js

```diff
@@ -211,6 +211,10 @@ export default class Core {
     }
   }
 
+  onControlsVisibilityChanged(isVisible) {
+    this.bridge.onVisibilityControlsChanged({ isVisible: isVisible });
```

Comment:
You can use shorthand here: `{ isVisible }` - strange that ESLint didn't mention that.

Reply:
done